.travis.yml

```diff
@@ -12,8 +12,24 @@ before_script:
 
 script:
     - cd $TRAVIS_BUILD_DIR
-    - echo "TODO need to build testing with samples"
-
+    - curl https://api.github.com/repos/vitasdk/autobuilds/releases  | grep browser_download_url | grep linux | head -n 1 | cut -d '"' -f 4 | xargs wget
```

Comment:
why mix curl + wget?
how about change to
`curl https://api.github.com/repos/vitasdk/autobuilds/releases  | grep browser_download_url | grep linux | head -n 1 | cut -d '"' -f 4 | xargs curl -L | tar xj` ?